downstream/modules/platform/proc-installing-containerized-aap.adoc

```diff
@@ -201,4 +181,17 @@ gateway_main_url=<https://load_balancer_url>
 [NOTE] 
 ====
 HAProxy SSL passthrough mode is not supported with platform gateway.
-====
\ No newline at end of file
+====
+
+.Installing containerized {PlatformNameShort}
+
+Use the following command to install containerized {PlatformNameShort}:
+
+----
+ansible-playbook -i inventory ansible.containerized_installer.install
+----
+
+* If your privilege escalation requires a password to be entered, append `-K` to the command line. You will then be prompted for the `BECOME` password. 
+* You can use increasing verbosity, up to 4 v's (`-vvvv`) to see the details of the installation process. However it is important to note that this can significantly increase installation time, so it is recommended that you use it only as needed or requested by Red Hat support.
```

Comment:
```suggestion
* You can use increasing verbosity, up to 4 v's (`-vvvv`) to see the details of the installation process. However, it is important to note that this can significantly increase installation time, so it is recommended that you use it only as needed or requested by Red Hat support.
```